test_conformance/basic/test_enqueued_local_size.cpp

```diff
@@ -74,6 +74,27 @@ test_enqueued_local_size(cl_device_id device, cl_context context, cl_command_que
     size_t localsize[2];
     int err;
 
+    // For a OpenCL-3.0 device this test should be skipped in the case that
+    // non-uniform work-groups are not supported.
+    if (get_device_cl_version(device) >= Version(3, 0))
+    {
+        cl_bool areNonUniformWorkGroupsSupported = false;
+        err = clGetDeviceInfo(device, CL_DEVICE_NON_UNIFORM_WORK_GROUP_SUPPORT,
+                              sizeof(areNonUniformWorkGroupsSupported),
+                              &areNonUniformWorkGroupsSupported, nullptr);
+        if (CL_SUCCESS != err)
+        {
+            test_error(err, "clGetDeviceInfo failed.");
```

Comment:
Should return `TEST_FAIL` here.

Reply:
Good catch, I've replace this whole block with `test_error_ret(err, "clGetDeviceInfo failed.", TEST_FAILED)". Thanks for the feedback!